specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/questionanswering-authoring.json

```diff
@@ -871,9 +1392,19 @@
       },
       "enum": [
         "json",
-        "qna"
+        "tsv",
+        "excel",
+        "allDataExcel",
+        "allDataTsv"
```

Comment:
These last couple aren't self-descriptive. Please considering adding documentation for the values in the `x-ms-enum` object property.

Reply:
Thanks, added descriptions.